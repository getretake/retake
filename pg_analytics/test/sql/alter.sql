CREATE TABLE t (a int, b text) USING deltalake;
ALTER TABLE t ADD COLUMN c int;
DROP TABLE t;
