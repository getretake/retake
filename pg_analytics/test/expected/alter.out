CREATE TABLE t (a int, b text) USING deltalake;
ALTER TABLE t ADD COLUMN c int;
ERROR:  ALTER TABLE is not yet supported. Please DROP and CREATE the table instead.
DROP TABLE t;
